src/com/musala/xml/model/School.java

```diff
@@ -0,0 +1,54 @@
+package com.musala.xml.model;
+
+import java.util.List;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement(name = "school")
+public class School {
+    private StudentListWrapper students;
+    private TeacherListWrapper teachers;
+    private List<SchoolClass> schoolClasses;
+    private String name;
+
+    public void addStudents(SchoolClass... schoolClasses) {
```

Comment:
Method is not used anywhere. Please remove any code snippets that are not in use.